.travis.yml

```diff
@@ -1,17 +1,32 @@
-language: cpp
+language: python
 dist: trusty
 sudo: required
 
-before_install:
-  - . .travis.setup
+stages:
+  - name: build
+  - name: deploy
+    if: tag =~ v.*
 
-before_script:
-  - mkdir build
-  - cd build
-  - cmake -G "Unix Makefiles" -DCMAKE_BUILD_TYPE=Release -DCUDA_TOOLKIT_ROOT_DIR=/usr/local/cuda-8.0 ..
+python:
+  - "3.4"
+
+env:
+  - CUDA_TOOLKIT_ROOT_DIR=/usr/local/cuda-8.0
+
+install:
+  - . .travis.install
 
 script:
-  - make -j2 VERBOSE=1
+  - python3 setup.py bdist_wheel
+  - auditwheel repair -w dist dist/*
+
+jobs:
+  include:
+    - stage: build
```

Comment:
I don't see much changes...

Also `script` should run the tests, not install.

And be aware that this project is actually C++. There are python bindings, but there can be other bindings too. Don't nail it to Python.